rubin_scheduler/scheduler/model_observatory/jerk.py

```diff
@@ -125,6 +127,8 @@ def _get_trajectory_instance_case(distance, v_max, acc_max, jerk_max):
     """Determine which motion profile to use.
 
     Modified from https://github.com/mdhom/py_constant_jerk/blob/main/constantJerk.py
+    which was probably derived from:
+    https://www.researchgate.net/publication/289374755_THIRD_ORDER_POINT-TO-POINT_MOTION_-PROFILE
```

Comment:
Did you see  mdhom's other repo here https://github.com/mdhom/PointToPointMotion ? That one is basically the C# version of the same thing, and includes the reference to the paper.

Reply:
Ah, I didn't see that one. I did put in a PR to the original python repo so see if they agree with the change.